src/assets/images/link_widget.svg

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 24.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Ebene_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 150 100" style="enable-background:new 0 0 150 100;" xml:space="preserve">
+<style type="text/css">
+	.st0{enable-background:new    ;}
```

Comment:
* [ ] FYI according to https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/enable-background `enable-background` is deprecated.

Reply:
@dcsaszar This is not my fault.. talk to Adobe.